intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -450,12 +741,14 @@ \subsection{Activating and de-activating modules}
 %\shellcmd{module purge}%
 \end{prompt}
 
-It is a good habit to use this command in job-scripts, prior to loading the
-modules specifically needed by applications in that job description. This
-ensures that no version conflicts occur if the user loads module using his
-'.bashrc' file.
+%TODO: really? it isn't in gent, we first load a module that sets modulepath
+% so module purge breaks further loading of modules
```

Comment:
actually, that's a bug we should fix, `module purge` should break the user's environment...